cabot/cabotapp/models.py

```diff
@@ -783,35 +793,36 @@ def _run(self):
         except requests.RequestException as e:
             result.error = u'Request error occurred: %s' % (e.message,)
             result.succeeded = False
+            return result, [type(e).__name__]  # use exception type as tag name (e.g. ConnectionError)
```

Comment:
Wonder if it's worth having an enum for tags (could do like  `enum: a = 'tag_name: {}'; tags = enum.a.format(info)` for ones that require input. Reasons would be

1) more schematized input--standardization across check types
2) if there's any characteristics the tags need to have (max length, unicode support, etc) we could just test the enum values for that--less risk of runtime failure

Let me know what you think, not a blocker for deploying

Reply:
I actually like that idea a lot...I'll go through and try that before deploying.

Reply:
I sorta compromised, for modularity - I added `tag_*` constants/helpers members to each check class. This makes it easy to see all the tags a check type uses.